internal/resource/obtenant/obtenant_task.go

```diff
@@ -608,3 +608,39 @@ func MaintainTenantParameters(m *OBTenantManager) tasktypes.TaskError {
 	}
 	return nil
 }
+
+func MaintainTenantVariables(m *OBTenantManager) tasktypes.TaskError {
+	variableMap := make(map[string]apitypes.Variable)
+	for _, variable := range m.OBTenant.Status.Variables {
+		m.Logger.V(oceanbaseconst.LogLevelDebug).Info("Build variable map", "variable", variable.Name)
+		variableMap[variable.Name] = variable
+	}
+	for _, variable := range m.OBTenant.Spec.Variables {
+		variableStatus, variableExists := variableMap[variable.Name]
+		if !variableExists {
+			m.Logger.V(oceanbaseconst.LogLevelDebug).Info("Variable not exists, need create", "variable", variable.Name)
+			err := m.createOBTenantVariable(&variable)
+			if err != nil {
+				m.Logger.Error(err, "Create obtenantvariable failed", "param", variable.Name)
```

Comment:
Maybe the `"param"` in the logging message should be replaced with `varaiable` or `var`